src/routes/controllers/articles.controller.js

```diff
@@ -11,6 +11,7 @@ import { FREE, DRAFT } from '../../utils/constants';
 import { calculateTimeToReadArticle } from '../../utils/readtime.utils';
 import { sendMailer } from '../../config/mail-config';
 
+import { createTag } from './tag.controller';
```

Comment:
i think ```tag``` here should read ```tags``` like every other controller in the codebase

Reply:
Ok, thanks. I will fix that.